Documentation/Blazorise.Docs/Pages/Docs/Components/Tooltips/TooltipPage.razor

```diff
@@ -125,4 +135,7 @@
     <DocsAttributesItem Name="AppendTo" Type="string" Default="body">
         The element to append the tooltip to. If <Code>Interactive = true</Code>, the default behavior is appendTo: "parent".
     </DocsAttributesItem>
+    <DocsAttributesItem Name="Delay" Type="(int Show, int Hide)" Default="(0, 0)">
```

Comment:
You don't usually like these at all, not even for internal things, what changed?

Reply:
Nothing much. I still prefer more complex objects over tuples. However, since this API will hardly change in the future, I find it appropriate in this case.